validator/src/cli/thread_args.rs

```diff
@@ -0,0 +1,43 @@
+//! Arguments for controlling the number of threads allocated for various tasks
+
+use {
+    clap::Arg, solana_clap_utils::input_validators::is_within_range,
+    solana_core::replay_stage::MAX_CONCURRENT_FORKS_TO_REPLAY,
```

Comment:
i think we just move the constant definition here

Reply:
No strong preference from me on this so did it in https://github.com/anza-xyz/agave/pull/236/commits/f117926c51a2f1019693a9b64af0d98236d74c5d